Makefile

```diff
@@ -13,7 +13,7 @@ DEIS_REGISTRY ?= ${DEV_REGISTRY}
 RC := manifests/deis-${SHORT_NAME}-rc.yaml
 SVC := manifests/deis-${SHORT_NAME}-service.yaml
 SEC := manifests/deis-${SHORT_NAME}-secretAdmin.yaml
-IMAGE := ${DEIS_REGISTRY}/${SHORT_NAME}:${VERSION}
+IMAGE := ${DEIS_REGISTRY}/${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
```

Comment:
Have we standardized?  No.  We're a long ways off from having a consistent dev workflow across all the projects, even though we all agree on the need for it.  I would guess it doesn't happen until post-alpha.  That said, at least two repos are already doing what I was suggesting here.  If not doing it, just be aware that you'll have troubles pushing CI-built images to docker hub, although quay.io won't be a problem.  I think we _are_ trying to push to both.